alephclient/api.py

```diff
@@ -353,6 +353,29 @@ def _bulk_chunk(
                     return
                 backoff(ae, attempt)
 
+    def write_entity(self, collection_id: str, entity: Iterable, **kw):
```

Comment:
One cute thing to do would actually be to define a type for `entity`, e.g. some sort of `Dict` alias? This would be used in `write_entities` but also the other methods that deal in `Iterable[EntityDict]`..